### PR TITLE
Added paradise city

### PR DESCRIPTION
**Pull request by brainhong913**

### CODE REVIEW

**Comment by claudiodsf** [comment]
Thanks for the pull request. As for the billboard format, please add a title to your article!

**Comment by brainhong913** [comment]
Added. Can you see I added the title here?

File: index.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
     <div class="column">
     </div>
     <div class="column">
+      <div id="example"><script>include("articles/paratise_city.md", "example")</script></div>
```
Comment (claudiodsf) [inline index.html]: Please replace `example` with the name of your article (in this case `paratise_city`). Do it in both places

File: articles/paradise_city.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Paradise city
```
Comment (claudiodsf) [inline articles/paradise_city.md]: Please use markdown syntax for title

File: articles/Paradise_city.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+**Paradise city**
```
Comment (claudiodsf) [inline articles/Paradise_city.md]: You have to use this syntax:

```
# Paradise city
```

Comment (brainhong913) [reply articles/Paradise_city.md]: done

**Comment by brainhong913** [comment]
Also should be resolved confilt

**Pull request merged**